src/Pretzel.Logic/Commands/ICommandParametersExtendable.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+using System.Composition;
+using System.Linq;
+using Pretzel.Logic.Extensibility;
+
+namespace Pretzel.Logic.Commands
+{
+    public interface ICommandParametersExtendable
```

Comment:
I am a little bit confused, what is this interface purpose?
It seems to me that this is not used.
If it is for extensibility, I don't think that everyone should touch at export factories, it should be simplified in another class or function.

Reply:
I agree on this one, I will refactor the code so it's not needed any longer.

Reply:
Not needed any longer